README.md

```diff
@@ -264,9 +264,11 @@ A new version will be published with each github release as it's part of Github
   - `release-major`                  updates version to next MAJOR version e.g. from 3.0.0 to 4.0.0
   - `release-minor`                  updates version to next MINOR version e.g. from 3.0.0 to 3.1.0
   - `release-patch`                  updates version to next PATCH version e.g. from 3.0.0 to 3.0.1
-- The make command will commit and push to github
-- Update README and CHANGELOG if necessary and commit the changes
+- The make command will commit and push to github (you have to fill in your GitHub username and password to log in)
```

Comment:
Probably you can extend the comment with:
`(you have to fill in your GitHub username and password to log in when not using a [github token](https://docs.github.com/en/enterprise/2.15/user/articles/creating-a-personal-access-token-for-the-command-line))`

Reply:
done in 83b3838